tokens/43114.ts

```diff
@@ -1,4 +1,12 @@
 export const AvalancheTokens = [
+    {
+    address: '0xfe07cab00DFB08C348De861FDDCBBdB9E9e49702',
+    decimals: 18,
+    name: 'dogwifhat',
+    symbol: 'WIF',
```

Comment:
Hey, could you confirm whether the symbol is WIF or $WIF?

Reply:
done ser. it is indeed $wif